apps/whale-api/src/module.api/poolpair.controller.ts

```diff
@@ -81,6 +84,8 @@ export class PoolPairController {
     const totalLiquidityUsd = await this.poolPairService.getTotalLiquidityUsd(info)
     const apr = await this.poolPairService.getAPR(id, info)
     const volume = await this.poolPairService.getUSDVolume(id)
+    info.nameTokenA = (await this.deFiDCache.getTokenInfo(info.idTokenA) as TokenInfo).name
+    info.nameTokenB = (await this.deFiDCache.getTokenInfo(info.idTokenB) as TokenInfo).name
```

Comment:
How about moving `mapPoolPair` function into the class function of `PoolPairController` so that you don't need to duplicate this code and instead have it embedded into the function to keep it DRY.

Reply:
Thanks for the suggestion! Updated.